target_athena/target.py

```diff
@@ -33,6 +33,7 @@ class TargetAthena(Target):
         th.Property("temp_dir", th.StringType),
         th.Property("stream_maps", th.ObjectType()),
         th.Property("stream_map_config", th.ObjectType()),
+        th.Property("athena_workgroup", th.StringType, default="primary"),
```

Comment:
What do you think of `athena_workgroup` vs `workgroup` vs `workgroup_name`?

I have no problem with `athena_workgroup` but wanted to see if you have any thoughts on removing the `athena_` prefix or if you considered any possible alternates.